garnett/fields.py

```diff
@@ -22,7 +24,7 @@ def __new__(cls, content, fallback=False, fallback_language=""):
         return instance
 
 
-def translation_fallback(field, obj):
+def translation_fallback(field: "TranslatedField", obj: Model) -> str:
```

Comment:
Do you think it is a good idea to move these functions to a field_utils file?

Reply:
Moved into issue